docs/configuration.md

```diff
@@ -291,13 +295,28 @@ Format for networks is:
 ...
 ```
 
+```json
+...
+"networks": {
+  "NETWORK NAME": {
+    "Host": "ADDRESS",
+    "NetworkKey": "ACCESS NODE NETWORK KEY"    
```

Comment:
Should this be lowercased now?

Reply:
https://github.com/onflow/flow-cli/pull/449/commits/05b28f3a8666def59064eebb64f01d7102dc1e6c